feature/session/src/main/res/layout/fragment_session_detail.xml

```diff
@@ -167,18 +167,17 @@
                     android:textIsSelectable="true"
                     app:layout_constraintEnd_toEndOf="parent"
                     app:layout_constraintStart_toStartOf="@id/session_title"
-                    app:layout_constraintTop_toBottomOf="@id/divider1"
+                    app:layout_constraintTop_toBottomOf="@id/divider_top_description"
                     tools:text="Material is the metaphor A material metaphor is the unifying theory of a rationalized space and a system of motion. The material is grounded in tactile reality, inspired by the study of paper and ink, yet technologically advanced and open to imagination and magic."
                     />
 
-
                 <ImageView
-                    android:id="@+id/divider2"
+                    android:id="@+id/divider_top_intended_audience"
                     android:layout_width="0dp"
                     android:layout_height="4dp"
                     android:layout_marginEnd="16dp"
                     android:layout_marginStart="16dp"
-                    android:layout_marginTop="28dp"
+                    android:layout_marginTop="20dp"
```

Comment:
Could you please explain why 8dp has gone just in case? 👀

Reply:
@jmatsu 
Because I thought margin was too wide compared to gallery 🤔

Reply:
make sense!